modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,285 @@
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster
+// * AWS account credentials
+* Configured
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+CLI (`rosa`)
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+[NOTE]
+====
+When installing an ALBO for use with
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+a ROSA
+endif::openshift-rosa[]
+cluster in an AWS Local Zone (LZ), the AWS LZ must be enabled for the account and AWS Elastic Load Balancing v2 (ELBv2) services must be available in the AWS LZ.
+====
+.Procedure
+
+. Identify the cluster infrastructure ID and the cluster OpenID Connect (OIDC) DNS by running the following commands:
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster INFRA ID:
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'Infra ID'
+----
++
+or
++
+[source,terminal]
+----
+$ oc get infrastructure cluster -o json | jq -r '.status.infrastructureName'
+----
+
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster OIDC DNS:
++
+--
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'OIDC'
+----
+
+Save the output from both commands. You will use this information in future steps within this procedure.
+
+--
+. Create and manage AWS Identity and Access Management (IAM) roles and policies:
++
+.. Create the AWS IAM policy required for the AWS Load Balancer Controller (ALBC) and the ALBO, which can be found link:https://github.com/openshift/aws-load-balancer-operator/blob/main/assets/iam-policy.json[here].
++
+.. Create an AWS IAM role and assign the AWS IAM policy displayed in step 2a to that role. For more information on creating AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html[Creating IAM roles]. For more information on adding AWS IAM policies to AWS IAM roles, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html[Adding and removing IAM identity permissions].
+
+.. Assign the following trust policy to the newly-created AWS IAM role. For more information on assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
+
++
+--
+[source,json]
+----
+{
+"Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "arn:aws:iam::{AWS-AccountNo}:<1>
+                                {Cluster's OIDC Endpoint}" <2>
```

Comment:
The format looks incorrect, please refer to https://github.com/openshift/aws-load-balancer-operator/pull/97/files
it should be like
```
{"Federated":"arn:aws:iam::123456789012:oidc-provider/oidc-abcd.s3.us-west-2.amazonaws.com"}
```

Reply:
Done.